### ISSUE TITLE
Intégrer les feedbacks d'Aurélia

### ISSUE THREAD

**Issue by GuillaumeAV** [assemblee-virtuelle/pair#133]
- Utiliser : Il s'agit d'une notion intéressante, différente de celle de consommer, et de moyen (intentions), qui permet notamment de décrire les usages qui sont faits de biens non-rivaux et donc non détruits, non consommés, lors de l'eur utlisation : Exemple, j'utilise un balai, j'utilise une base de connaissance)

- Détruire : Concept non présent mais pourtant intéressant, qui pourrait permettre de visibiliser les processus de destruction en complément des processus de création. Cf *Communs négatifs* 

- Externalités positives & négatives : Sont elle présentes dans l'ontologie telle qu'elle est à) l'heure actuelle ? Réflexion intéressante à mener !!

- Biens naturels, avec leurs propriétés spécifiques : Autopoiesis notamment. Devrait-on les intégrer ??

**Comment by tfrancart** [comment]
>  Utiliser : Il s'agit d'une notion intéressante, différente de celle de consommer, et de moyen (intentions), qui permet notamment de décrire les usages qui sont faits de biens non-rivaux et donc non détruits, non consommés, lors de l'eur utlisation : Exemple, j'utilise un balai, j'utilise une base de connaissance)

On l'a, c'est la relation `uses`, différente de la relation `consumes`; pour faire une pizza je `consumes` de la farine qui se retrouve dans le produit final mais je `uses` un four à pizza qui me permet de cuire la pizza mais qui ne se retrouve pas dans le produit final.

**Comment by GuillaumeAV** [comment]
Oui, l'issue portait davantage sur les notions de destruction et d'externalités qu'on pourrait envisager d'intégrer, ce qui pourrait générer plein de cas d'usages en lien avec la RSE, l'éco circulaire etc.
